cf-custom-resources/lib/nlb-cert-validator-updater.js

```diff
@@ -319,6 +324,107 @@ exports.deadlineExpired = function () {
     });
 };
 
+/**
+ * Load clients and variables that can be reused between calls.
+ */
+function loadResources() {
+    if (!acm) {
+        acm = new AWS.ACM();
+    }
+
+    if (!envRoute53) {
+        envRoute53 = new AWS.Route53();
+    }
+
+    domainTypes = {
+        EnvDomainZone: {
+            regex: new RegExp(`^([^\.]+\.)?${envName}.${appName}.${domainName}`),
+            domain: `${envName}.${appName}.${domainName}`,
+        },
+        AppDomainZone: {
+            regex: new RegExp(`^([^\.]+\.)?${appName}.${domainName}`),
+            domain: `${appName}.${domainName}`,
+        },
+        RootDomainZone: {
+            regex: new RegExp(`^([^\.]+\.)?${domainName}`),
+            domain: `${domainName}`,
+        },
+        OtherDomainZone: {},
```

Comment:
I'm just curious about this domainType. How is it used?

Reply:
Good catch! This is not used. I should just delete it!